epub33/rs/index.html

```diff
@@ -1170,24 +1146,124 @@ <h2>Open Container Format Processing</h2>
 				<h3>OCF Abstract Container</h3>
 
 				<section id="sec-container-iri">
-					<h4>Relative URLs for Referencing Other Components</h4>
+					<h4>URL of the Root Directory</h4>
 
-					<p>For <a href="https://url.spec.whatwg.org/#relative-url-with-fragment-string"
-							>relative-URL-with-fragment strings</a> [[URL]], Reading Systems MUST determine the <a
-							href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]] according to the
-						relevant language specifications for the given file formats. For example, CSS defines how
-						relative URL references work in the context of CSS style sheets and property declarations
-						[[CSSSnapshot]].</p>
+					<p>Reading Systems MUST assign a URL [[URL]] to the <a>Root Directory</a> of the <a>OCF Abstract Container</a>. This URL is called the <a data-cite="epub-33#dfn-container-root-url">container root URL</a>. The URL itself is implementation specific, but:</p>
+
+					<ul>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing </a> [[URL]] "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url">base</a> MUST be the <a>container root URL</a>.</li>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing </a> [[URL]] "<code>..</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url">base</a> MUST be the <a>container root URL</a>.</li>
+						<li>The <a data-cite="url#origin">origin</a> of the <a>container root URL</a> MUST be unique for each EPUB Publication instance.</li>
+						<li>the <a data-cite="url#origin">origins</a> of the <a data-cite="epub-33#dfn-container-url">container URLs</a> of all the files in the <a>OCF Abstract Container</a> MUST be identical.</li>	
+					</ul>
+
+					<p class="note">The unicity of the <a href="https://url.spec.whatwg.org/#origin">origin</a> per EPUB Publication
+						instance means that if two different users acquire a copy of the same EPUB Publication, the
```

Comment:
I see you left a note below about EPUB instance. It think we will have a difficult time defining that, but without a proper definition this isn't a very useful paragraph. On the other hand, 'tis just a note, so maybe I shouldn't worry so much.

Reply:
Yes, you shouldn't worry so much :-)

Reply:
Yes, I think you shouldn't worry too much :-)